community-operators/jenkins-operator/0.7.0/manifests/jenkins-operator.v0.7.0.clusterserviceversion.yaml

```diff
@@ -4,104 +4,112 @@ metadata:
   annotations:
     operators.openshift.io/infrastructure-features: '["Disconnected"]'
     alm-examples: |-
-      [{
-        "apiVersion": "jenkins.io/v1alpha2",
-        "kind": "Jenkins",
-        "metadata": {
-          "generation" : 1,
-          "name" : "example"
-        },
-        "spec": {
-          "backup": {
-            "action": {},
-            "containerName": "",
-            "interval": 0,
-            "makeBackupBeforePodDeletion": false
-          },
-          "configurationAsCode": {
-            "enabled": true,
-            "defaultConfig": true,
-            "enableAutoReload": true,
-            "configurations": []
+      [
+        {
+          "apiVersion": "jenkins.io/v1alpha2",
+          "kind": "Jenkins",
+          "metadata": {
+            "name": "jenkins-sample"
           },
-          "jenkinsAPISettings": {
+          "spec": {
+            "configurationAsCode": {
+              "configurations": null,
+              "defaultConfig": true,
+              "enableAutoReload": true,
+              "enabled": true
+            },
+            "jenkinsAPISettings": {
               "authorizationStrategy": "serviceAccount"
-          },
-          "clusterRoles": [
-            {
-              "apiGroup": "rbac.authorization.k8s.io",
-              "kind": "ClusterRole",
-              "name": "edit"
+            },
+            "jenkinsImageRef": "jenkinsimage-sample",
+            "master": {
+              "containers": [
+                {
+                  "image": "jenkins/jenkins:lts",
```

Comment:
If we specify an image here different from the default jenkins image, it will be taken in account
The issue is that the lts image doesnt contain the prerequisite plugins (configuration as code, kubernetes ...)
Please specify here the image same as the default one containing these plugins